plugins/drng/drng.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	// Pollen defines the instance ID of the Pollen drng committee.
+	// Pollen defines the instance ID of the Goshimmer drng committee.
```

Comment:
here and in general for the entire dRNG we need to think of a better name for the committee

Reply:
ok reverted